TUM-Dev/NavigaTUM
# `calendar`-`data` dependency reduction

Previously, when deploying data, a redeployment of the calendar was necessary.
This is part of the process of making this a StatefullSet

## Proposed Changes (include Screenshots if possible)

- Added a new API to the main-api (`/internal/list/ids_with_calendar`)
- Migrated the 

## How to test this PR

1. Launch server
2. in another terminal set 
   `API_SVC_SERVICE_HOST=localhost`
   `API_SVC_SERVICE_PORT_API=8080`
   run the calendar
3. See that the calendar comes up correctly.

## How has this been tested?

- as stated above

## Checklist:
